pkg/tfgen/generate.go

```diff
@@ -986,3 +986,16 @@ func generateManifestDescription(info tfbridge.ProviderInfo) string {
 	return fmt.Sprintf("%s. Based on terraform-provider-%s: version v%s", info.Description, info.Name,
 		info.TFProviderVersion)
 }
+
+func getLicenseTypeURL(license tfbridge.TFProviderLicense) string {
+	switch license {
+	case tfbridge.MITLicenseType:
+		return "https://mit-license.org/"
+	case tfbridge.MPL20LicenseType:
+		return "https://www.mozilla.org/en-US/MPL/2.0/"
+	case tfbridge.Apache20LicenseType:
+		return "https://www.apache.org/licenses/LICENSE-2.0.html"
+	default:
+		return ""
```

Comment:
```suggestion
		contract.Failf("Unrecognized license: %v", license)
```

?

Reply:
good idea - I added it above the return "", it needs to have return "" otherwise it won't build. But it will fail before it gets there :)